src/pages/EmergencyViewPage/EmergencyViewPage.scss

```diff
@@ -0,0 +1,317 @@
+@import "scss/constants.scss";
```

Comment:
Overall file comments:

- Duplicate styles need to be extracted (together with duplicates in `tsx` file)
- `px`, `%` and other literal values needs to be extracted into constants. If similar alread exist in `constants.scss` those should be picked.
- naming should be changed to conform to BEM convention:
  * `-` character as a separator between words
  * `__` as element indicator
  * `--` as mofificator indicator.